Feature/adaptive

KseniyaSokolovskaya opened the pull request:
21 june 2019
fix: posts for 992px - 1200px;

20 june 2019
feat: add responsive.scss;  …
ref: split common.scss into components;

fix:
font for paralax-section, service;
padding,border,color for post-item;
delete border in portfolio;
button-to-up;
zoom and link icons;

ref:
replace button into link in post-item;
divaider to divider;
add list for social links;
delete list in footer;

feat: 
add breakpoints and mixins for responsive, add properties for screen-xl-large, screen-x-large;

KseniyaSokolovskaya committed:
feat: add responsive.scss;
ref: split common.scss into components;

KseniyaSokolovskaya committed:
fix:
font for paralax-section, service;
padding,border,color for post-item;
delete border in portfolio;
button-to-up;
zoom and link icons;

ref:
replace button into link in post-item;
divaider to divider;
add list for social links;
delete list in footer;

KseniyaSokolovskaya committed:
add breakpoints and mixins for responsive, add properties for screen-xl-large, screen-x-large

angelinabylitskaya commented on scss/components/_blog.scss:
```diff
@@ -0,0 +1,182 @@
+    @include screen-x-large {
+      @include arrow-left ($color-arrows-base, 15px, 3px);
+    }     
+    top: 41%;
```
Bad values. Do not use % here

angelinabylitskaya commented on scss/components/_blog.scss:
```diff
@@ -0,0 +1,182 @@
+    word-spacing: 0px;
+    color: $color-txt-base;
+  }
+  .slidershow {
```
Code style: empty line before each selector

angelinabylitskaya commented on scss/components/_menu.scss:
```diff
@@ -0,0 +1,49 @@
+  padding: 0px 100px;
+  
+
+  a {
```
Better to use classes instead of tags in css

angelinabylitskaya commented on scss/components/_slideshow-section.scss:
```diff
@@ -0,0 +1,5 @@
+  @include dam($color-accent);
+  background-color: $color-primary;
+  position: relative;
+}
```
Add an empty line at the end of the files

angelinabylitskaya commented on scss/components/_blog.scss:
```diff
@@ -0,0 +1,182 @@
+  }
+  .slidershow {
+    margin-bottom: 96px;
+    @include screen-x-large {
```
Consider inserting an empty line before includes with content

angelinabylitskaya commented on scss/components/_blog.scss:
```diff
@@ -0,0 +1,182 @@
+    padding: 219px 25px 0;
+    text-transform: capitalize;
+    width: calc((100% - 90px)/3);
+    max-width: 370px;
```
Do not use max-width here. Item should be adaptive

angelinabylitskaya commented on scss/components/_blog.scss:
```diff
@@ -0,0 +1,182 @@
+    }
+
+    &_last {
+      margin-right: 0 !important;
```
Do not use !important

angelinabylitskaya commented:
![image](https://user-images.githubusercontent.com/19670773/59879439-638b1880-93b3-11e9-828f-a9938062d9e3.png)

KseniyaSokolovskaya committed:
ref:
position arrows( replace % on function calc);
add empty line before selector;
replace tag selector on class selector;

KseniyaSokolovskaya committed:
fix: posts for 992px - 1200px
